src/functions.js

```diff
@@ -88,6 +140,14 @@ function paintShop( cars, colour ){
 
   // the original array passed in should not change
   // hint: look up 'Cloning objects in JavaScript'
+  var copy = [];
+  for (i = 0; i < cars.length; i++) {
+    copy[i] = Object.assign({}, cars[i]);
+    if (copy[i].make === 'Ford') {
+      copy[i].colour = colour;
+    };
+  };
+  return copy;
 }
```

Comment:
Whilst this one works, give some thought to what would happen if the test was to use a different make of vehicle. Hard coding values into a function is rarely a good idea, as it limits their reuse, which is a functions' main purpose. I'd be happy to take you through a different (reusable) solution tomorrow.